### ISSUE TITLE
Add support for importing aws_autoscaling_policy

### ISSUE THREAD

**Issue by Szasza** [terraform-providers/terraform-provider-aws#6578]
### Description

If I try to import an `aws_autoscaling_policy` I get the error message saying `resource aws_autoscaling_policy doesn't support import`. This makes moving existing infrastructure into Terraform slightly more cumbersome.

It would be ideal to provide import support for this resource type.

### New or Affected Resource(s)

* aws_autoscaling_policy

### Potential Terraform Configuration

The Terraform command for example could be

```hcl
terraform import aws_autoscaling_policy.SOMEIDHERE asg-name/scale-up
```

### References

* The respective [AWS CLI command](https://docs.aws.amazon.com/cli/latest/reference/autoscaling/put-scaling-policy.html) identifies a scaling policy based on the ASG name and the policy name.

**Comment by Szasza** [comment]
Also happy to contribute.

**Comment by bflad** [comment]
Support for `aws_autoscaling_policy` resource import has been merged in and will release with version 1.57.0 of the Terraform AWS provider, likely later today. 👍

**Comment by bflad** [comment]
This has been released in [version 1.57.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1570-january-26-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
